app/experimenter/experiments/api/v5/serializers.py

```diff
@@ -821,9 +821,7 @@ class Meta:
         ]
 
     def get_feature_configs(self, obj):
-        return ", ".join(
-            [feature.name for feature in obj.feature_configs.order_by("name")]
-        )
+        return ", ".join([feature.name for feature in obj.feature_configs.all()])
```

Comment:
Oh yes this is almost right, querying against .all() will use the cached queryset, but it was ordered by name so that it's stable in tests, so we still have to order but we can do it in memory with `sorted`